### DISCUSSION TITLE
Release v1.0.15 / 5.55.15

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1725] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Fixed
- fixed memory corruption introduced in the last build causing Chrome to crash sometimes
- FIXED SECURITY ISSUE: NtCreateSymbolicLinkObject was not filtered (thanks Diversenok)

### Added
- added notification to warn that the default update checker is lagging behind the newest release on GitHub, to ensure that only bug-free builds are offered as updates [#1682](https://github.com/sandboxie-plus/Sandboxie/issues/1682)
- added main browsers to BlockSoftwareUpdaters template (by Dyras) [#1630](https://github.com/sandboxie-plus/Sandboxie/pull/1630)
- added a warning when Sandboxie-Plus.ini is not writeable [#1681](https://github.com/sandboxie-plus/Sandboxie/issues/1681)
- added clean-up for critical sections (by chunyou128) [#1686](https://github.com/sandboxie-plus/Sandboxie/pull/1686)

### Changed
- improved command line handling for breakout processes [#1655](https://github.com/sandboxie-plus/Sandboxie/issues/1655)
- disabled SBIE2193 notification (by isaak654) [#1690](https://github.com/sandboxie-plus/Sandboxie/pull/1690)
- improved error message 6004 [#1719](https://github.com/sandboxie-plus/Sandboxie/issues/1719)

### Fixed
- fixed dark mode issue with the new tray list
- fixed not showing a warning when Sandboxie-Plus.ini is not writeable [#1681](https://github.com/sandboxie-plus/Sandboxie/issues/1681)
- fixed issue with software compatibility checkbox (thanks MitchCapper) [#1678](https://github.com/sandboxie-plus/Sandboxie/issues/1678)
- fixed issue with events on box closure not always being executed [#1658](https://github.com/sandboxie-plus/Sandboxie/issues/1658)
- fixed memory leaks in key_merge.c
- fixed issue enumerating registry keys in privacy mode
- fixed settings issue introduced in 1.0.13 [#1684](https://github.com/sandboxie-plus/Sandboxie/issues/1684)
- fixed crash issue when parsing firewall port options
- FIXED SECURITY ISSUE: in certain cases a sandboxed process could obtain a handle on an unsandboxed thread with write privileges [#1714](https://github.com/sandboxie-plus/Sandboxie/issues/1714)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.15'>Release v1.0.15 / 5.55.15</a>.</em>

**Comment by BilleBarrett** [comment]
installed this update on several computers
had to reboot and after that it seems to be running OK
Had an older laptop, that I had to do a clean uninstall and reboot then install
OK

**Comment by BilleBarrett** [comment]
Question where can I find the trace files that are enabled.

**Reply by isaak654** [reply]
In case you meant the trace log feature of SBIE Plus: [TraceLog](https://sandboxie-plus.github.io/sandboxie-docs/Content/TraceLog.html)

**Comment by BilleBarrett** [comment]
The client sent me an email error from Sandboxie 5.55.15 
SBIE 2111. The process is not accessible.$ explorer.exe call Open Process (C00000022) access =00000040 initialized =1
Not sure what to tell him or if there is box left un checked

**Reply by isaak654** [reply]
The error message SBIE 2111 was introduced in [v5.55.17 / v1.0.17](https://github.com/sandboxie-plus/Sandboxie/commit/32132affb6c088336a6e83d19e04851a2840c2e0#diff-d7d583c391d8b7aa28609e54893ae147b9720faa68f79bc13e9ff41f510cddec), I don't see how v5.55.15 could be involved.

**Comment by BilleBarrett** [comment]
Yes, he read my email about backing up 17 to 15 and he assumed he has 15.
I just updated him back to 15 he’s ok now

 

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.***
Is your computer up to date and secure “NO” lets talk
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
